Fix type params tuple error (#4684)
psf/black: CHANGES.md

- Handle `# fmt: skip` followed by a comment at the end of file (#4635)
- Fix crash when a tuple appears in the `as` clause of a `with` statement (#4634)
- Fix crash when tuple is used as a context manager inside a `with` statement (#4646)

> - Fix crash when parenthesis are inside a type annotation (#4684)

Reviewer: ```suggestion
- Fix crash on parenthesized expression inside a type parameter bound (#4684)
```
A TypeVar bound is not technically a type annotation
